src/middlewares/login.js

```diff
@@ -0,0 +1,27 @@
+import jwt from 'jsonwebtoken';
+
+const isLoggedIn = async (req, res, next) => {
+  let token = req.headers['x-auth-access'];
+  if (!token) {
+    return res.status(401).json({
+      status: 'error',
+      message: 'You are not loggged in'
+    });
+  }
+  try {
+    token = token.slice(7);
```

Comment:
I think you can split by space `' '` and use [1] to get the token as opposed to slice. Just a suggestion though.

Reply:
Done. Thanks